### ISSUE TITLE
List selection

### ISSUE THREAD

**Issue by joshuadavidlee** [OrchardCMS/OrchardCore#2299]
I can not see how you add content items to a list in the editor.  In Orchard CMS there was a popup dialog and I could select one or many items.  I don't see that in Orchard Core.

**Comment by sebastienros** [comment]
You can't add existing ones, just add new ones right now.

**Comment by JoshTango** [comment]
but in the future we can add existing ones?  If so how many sprints away?

**Comment by sebastienros** [comment]
That's what list is doing. But in this case you might not need the List content type, just a field that will reference other content items, and this way we can keep the List as it is. This will be available soon I assume as we have someone already working on it.

**Comment by JoshTango** [comment]
Yes I noticed there is a content picker  field in the old version but  the content items were displayed horizontally rather than displayed in a nice list.  Maybe the new core version will require making a template for how the children are  displayed

**Comment by JoshTango** [comment]
Content picker field will not work for my needs.  I need to pick existing items and be able to order them how I want.  In the list so far I can not see how to pick items.

**Comment by deanmarcussen** [comment]
see also https://github.com/OrchardCMS/OrchardCore/issues/3667
